openmdao/error_checking/check_config.py

```diff
@@ -165,6 +172,11 @@ def _get_used_before_calc_subs(group, input_srcs):
             oparts = src_abs.split('.')
             src_sys = oparts[glen]
             tgt_sys = iparts[glen]
+            if (src_sys in parallel_connections and tgt_sys in parallel_connections and
+                    (parallel_connections[src_sys] != "NL: NLBJ")):
+                simple_warning("Need to attach NonlinearBlockJac to '%s' when connecting "
```

Comment:
Strictly speaking, couldn't it be another nonlinear solver? Would Newton not work in this case?

Reply:
Added Newton and Broyden